ocs_ci/deployment/mce.py

```diff
@@ -337,3 +338,32 @@ def wait_mce_resources(self):
                 **depl_ocp_obj.get(retry=60, wait=10, dont_raise=True)
             )
             deployment_obj.wait_for_available_replicas(timeout=600)
+
+    def get_installed_mce_version(self):
+        """
+        ##### TO DO ######
+        """
```

Comment:
I don't think that there should be a function without implementation.

Reply:
Have removed